docker/mysql/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM debian:jessie
```

Comment:
It's better to fix base image version here. So build will be reproducible next time in future.
like `debian:jessie:12.2`